Doc/tools/templates/download.html

```diff
@@ -12,6 +12,20 @@
   {% set dl_version = release %}
 {% endif %}
 
+{%- block extrahead -%}
+    {{ super() }}
+    <meta property="og:title" content="Download — Python {{ version }} documentation">
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="https://docs.python.org/{{ version }}/download.html">
```

Comment:
`og:url` is the canonical URL, so we always use `/3/`, even for versioned documentation.
```suggestion
    <meta property="og:url" content="https://docs.python.org/3/download.html">
```

Reply:
I find this confusing. If we’re looking at 3.13 documentation, the canonical URL using 3.13 makes it permanently useful, whereas the meaning of 3 changes over time.

Reply:
It's for SEO: each page has a `rel="canonical"` tag for the same reason (e.g. `<link rel="canonical" href="https://docs.python.org/3/library/index.html" />`). It is a little confusing but the best approach we have currently.